src/aliceVision/sfmData/View.cpp

```diff
@@ -17,18 +17,45 @@
 namespace aliceVision {
 namespace sfmData {
 
-float View::getCameraExposureSetting() const
+float View::getCameraExposureSetting(const float referenceISO, const float referenceFNumber) const
 {
     const float shutter = getMetadataShutter();
     const float fnumber = getMetadataFNumber();
+    const float iso = getMetadataISO();
+    
     if(shutter < 0 || fnumber < 0)
         return -1.f;
 
-    const float iso = getMetadataISO();
-    const float isoRatio = (iso < 0.f) ? 1.0 : (iso / 100.f);
+    float lReferenceIso = referenceISO;
+    if (lReferenceIso < 0.0f) {
```

Comment:
We should do something is "iso" is -1.
If "iso" or "referenceISO" or both are -1, we should put both to the same value to neutralize it.